src/core/server/services/comments/pipeline/phases/staff.ts

```diff
@@ -12,7 +12,13 @@ export const staff: IntermediateModerationPhase = ({
   author,
   now,
 }): IntermediatePhaseResult | void => {
-  if (author.role !== GQLUSER_ROLE.COMMENTER) {
+  const staffRoles = [
+    GQLUSER_ROLE.ADMIN,
+    GQLUSER_ROLE.MODERATOR,
+    GQLUSER_ROLE.STAFF,
+  ];
+
+  if (staffRoles.some(r => r === author.role)) {
```

Comment:
Feel like it's better if we clarify this since now an EXPERT is a thing... so we want to be more specific.

Let me know if there is a helper method/constant list we could use instead?